src/main/java/mate/jdbc/service/AbstractService.java

```diff
@@ -0,0 +1,15 @@
+package mate.jdbc.service;
+
+import java.util.List;
+
+public interface AbstractService<T> {
```

Comment:
It’s a good solution for the dao layer but not for services

Reply:
thanks, I got it